spec/requests/authentication_spec.rb

```diff
@@ -11,4 +11,11 @@
       end
     end
   end
+
+  context "attempting to visit a public path" do
+    it "does not require basic auth" do
```

Comment:
I'll have a dig into this. @RMcVelia seemed pretty sure that basic auth was blocking them before and we'd have to drop it in this case.

Reply:
Don't the non-prod envs have basic-auth, so it would need to be off for the metrics endpoint? Could you deploy a review app for this PR, as it would be good to see the metrics being generated?

Reply:
@stevehook @RMcVelia I've added a way to whitelist certain paths for all envs and made `/metrics` the first of them. See [this commit](https://github.com/DFE-Digital/register-trainee-teachers/pull/4460/commits/5a8fb1d0aa391d5adf055cf0e68704880ff2064e) and let me know what you think. Changes are now being deployed to [review-4460](https://register-pr-4460.test.teacherservices.cloud/) Rick 👍